libs/providers/flipt-web/src/lib/context-transformer.ts

```diff
@@ -1,4 +1,4 @@
-import { EvaluationContext } from '@openfeature/server-sdk';
```

Comment:
we we're importing from server in a web lib :fearful: 

This was caught because the new lint tried to add the server-sdk to the deps.